common/src/main/java/xyz/wagyourtail/jsmacros/client/gui/screens/EventMacrosScreen.java

```diff
@@ -24,8 +24,6 @@ protected void init() {
         super.init();
         eventScreen.setColor(0x4FFFFFFF);
         
-        keyScreen.onPress = (btn) -> this.openParent();
```

Comment:
huh. this is to prevent recursive key->event->key structure in the parents. is this fixed somewhere else?

Reply:
I promoted the service button to the top and thus moved the common code from all 3 Screens into the MacroScreen class.